helpers/test-helper.js

```diff
@@ -414,13 +414,16 @@ export async function getActiveRepCycle(colonyNetwork) {
   return repCycle;
 }
 
-export async function advanceMiningCycleNoContest(colonyNetwork, test, miningClient = undefined) {
+export async function advanceMiningCycleNoContest(colonyNetwork, test, miningClient = undefined, minerAddress) {
   await forwardTime(MINING_CYCLE_DURATION, test);
   const repCycle = await getActiveRepCycle(colonyNetwork);
+
   if (miningClient !== undefined) {
     await miningClient.submitRootHash();
   } else {
-    await repCycle.submitRootHash("0x00", 0, 10);
+    const accounts = await web3GetAccounts();
+    minerAddress = minerAddress || accounts[5]; // eslint-disable-line no-param-reassign
+    await repCycle.submitRootHash("0x00", 0, 10, { from: minerAddress });
```

Comment:
Can we just default to `accounts[5]` and omit the argument? That's essentially what we had before when we defaulted to `accounts[0]`.

Reply:
`repCycle` is a truffle contract and so we can't intercept this and have a "default". That plus I'd like us to understand who's calling this function.

Reply:
For the record, I think you can set defaults on `truffle-contract` instances, but I prefer making it explicit here too.

Reply:
`Explicit is better than implicit`, or so they keep telling me.